File: (X) Adicionales/(X.2) Práctica para laboratorios/(X.2.2) Ejercicios de C/G1/1-2.c
/* Escribir un programa en C que, dado dos números enteros, realice la suma,
resta, multiplicación, división entera y resto de los mismos. Los números
deben ser ingresados por el usuario. */

#include <stdio.h>

int main(void) {
    int x, y;

    printf("Ingrese un valor para X: ");
    scanf("%d", &x);
    printf("Ingrese un valor para Y: ");
    scanf("%d", &y);

    int suma = x + y;
    int resta = x - y;
    int mult = x * y;
    int div_ent = x / y;
    int resto = x % y;

    printf("\n");
    printf("Valor de X: %d\n", x);
    printf("Valor de Y: %d\n", y);

    printf("\n");
    printf("Resultado de la suma: %d\n", suma);
    printf("Resultado de la resta: %d\n", resta);
    printf("Resultado de la multiplicación: %d\n", mult);
    printf("Resultado de la división entera: %d\n", div_ent);
    printf("Resultado del resto: %d\n", resto);

    return 0;
}

File: algoritmos/(X) Adicionales/(X.2) Práctica para laboratorios/(X.2.1) Ejercicios de Pascal/G1/1-2.pas
{* Escribir un programa en pascal que dado dos números enteros realice la
suma, resta, multiplicación, división entera y resto de los mismos. Tanto
X como Y deben ser ingresados por el usuario. *}

program EJ2;

var
    x, y: integer;
    suma, resta, mult, div_ent, resto: integer;

begin
    write('Introduzca un valor para X: ');
    readln(x);

    write('Introduzca un valor para Y: ');
    readln(y);

    suma := x + y;
    resta := x - y;
    mult := x * y;
    div_ent := x div y;
    resto := x mod y;

    writeln();
    writeln('Valor de X: ', x);
    writeln('Valor de Y: ', y);

    writeln();
    writeln('Suma: ', suma);
    writeln('Resta: ', resta);
    writeln('Multiplicación: ', mult);
    writeln('División entera: ', div_ent);
    writeln('Resto: ', resto);
end.